dev/reference/alert-rules.md

```diff
@@ -0,0 +1,1274 @@
+---
+title: TiDB Cluster Alert Rules
+summary: Learn the alert rules in a TiDB cluster.
+category: reference
+---
+
+<!-- markdownlint-disable MD024 -->
+
+# TiDB Cluster Alert Rules
+
+This document describes the alert rules for different components in a TiDB cluster, including the rule descriptions and solutions of the alert items in TiDB, TiKV, PD, TiDB Binlog, Node_exporter and Blackbox_exporter.
+
+## TiDB alert rules
+
+This section gives the alert rules for the TiDB component. Each alert rule is classified according to their severity level - emergency, critical or warning.
```

Comment:
To make the severity level relationship clearer.

```suggestion
This section gives the alert rules for the TiDB component. According to the severity level, alert rules can be divided into three categories (from high to low): emergency-level, critical-level, warning-level.
```

Reply:
Updated. PTAL again. @lilin90